tests/osaka/eip7692_eof_v1/eip5450_stack/test_code_validation.py

```diff
@@ -561,9 +561,67 @@ def test_all_opcodes_stack_underflow(
                 ),
             ],
         ),
+        Container(
+            name="underflow_variable_stack_3",
+            sections=[
+                Section.Code(
+                    Op.RJUMPI[8](0)
+                    + Op.PUSH0 * 2
+                    + Op.RJUMPI[1](0)
+                    + Op.POP * 2
+                    + Op.PUSH0 * 2
+                    + Op.REVERT,
+                    max_stack_height=4,
```

Comment:
```suggestion
                    max_stack_height=3,
```
I think 4 is never reached, so in case a faulty client misses the underflow check, container validation could still fail due to the invalid max stack height.